stable_baselines/ddpg/ddpg.py

```diff
@@ -872,11 +873,12 @@ def learn(self, total_timesteps, callback=None, log_interval=100, tb_log_name="D
                             # Randomly sample actions from a uniform distribution
                             # with a probabilty self.random_exploration (used in HER + DDPG)
                             if np.random.rand() < self.random_exploration:
-                                rescaled_action = action = self.action_space.sample()
+                                unscaled_action = self.action_space.sample()
+                                action = scale_action(self.action_space, unscaled_action)
```

Comment:
Maybe a comment here would be good to explain why we are doing so ;)

Reply:
Added to all 3